### DISCUSSION TITLE
How to best check list of stickers for listing in signalstickers.com?

### DISCUSSION THREAD

**Discussion by mziehm** [signalstickers/signalstickers#861] [Ideas / feature request]
How can I best check a list of, e.g 400, signal.art links if they are included in signalstickers.com?
e.g. Is there are list of stickers include in signalstickers.com?

Obviously a checking option similar to https://signalstickers.com/contribution-status but allowing to enter a list of URL and returning a table with URL and status, would be a great and convenient solution, but I am open to anything.

**Comment by romainricard** [comment]
Do you know a bit of Python ? If so, you can use the /packs/ API endpoint, and parse the JSON (https://api.signalstickers.com/v1/packs/). Let me know if I can help!

**Comment by mziehm** [comment]
Yes, that's a good solution. Thanks. I'll give it a go. I didn't realize there was a API.

**Comment by romainricard** [comment]
I can also provide tools less painful than a Web interface if you need to propose a large amont of packs. Let me know !

**Reply by mziehm** [reply]
Yes please, that might be useful.

**Comment by mziehm** [comment]
I checked this source https://github.com/signal-stickers/signal-stickers.github.io/blob/master/packs.json
Of the ~1100 Packs, less than 30 are not contained in signalstickers ([signal-sticker-io-packs_new.json.txt](https://github.com/signalstickers/signalstickers/files/7653837/signal-sticker-io-packs_new.json.txt)). Most or all could be related to #863. Haven't checked them further (e.g. for duplicates or other exclusion reasons)

**Comment by romainricard** [comment]
Yep, this is probably why they are not on signalstickers, I don't remember specifically what happened when I added these stickers. I'll check https://github.com/signalstickers/signalstickers/issues/863, thanks!
